pkg/consensus/consensus.go

```diff
@@ -121,6 +122,24 @@ func NewService(cfg Config) (Service, error) {
 		return nil, err
 	}
 
+	// Check that wallet password is correct.
+	vals, err := cfg.Chain.GetValidators()
```

Comment:
I'm at 0, I'm not yet elected, still I have a password for some account and it should work.

Reply:
This is where the check in `getKeyPair` comes into play.

Reply:
Checking all available accounts is not an option, because password is per-account, not per-wallet.

Reply:
Well, we can check that password match to at least one account.